corehq/apps/app_manager/tests/data/suite/child-module-form-workflow-previous.xml

```diff
@@ -26,11 +26,11 @@
     </session>
     <stack>
       <create>
-        <command value="'m0'"/>
-        <command value="'m1'"/>
         <datum id="case_id" value="instance('commcaresession')/session/data/case_id"/>
         <datum id="case_id_new_guppy_0" value="uuid()"/>
-        <command value="'m1-f0'"/>
+        <command value="'m0'"/>
+        <datum id="case_id_guppy" value="instance('commcaresession')/session/data/case_id_guppy"/>
```

Comment:
This datum was missing from this test, which I added in https://github.com/dimagi/commcare-hq/pull/16797/

I think this is just wrong, and this test app probably didn't fully mimic the real broken form fixed by that PR (which no longer exists). I created a local app that matched the one used in this test, and in app preview previous screen behaved incorrectly and was then fixed by this PR.